helm/cluster-api-provider-aws/values.yaml

```diff
@@ -3,18 +3,27 @@ infrastructure:
   image:
     name: giantswarm/cluster-api-aws-controller
     tag: 787579502a18152ab2c1f99b7b460e60c607b7ab
+eks:
+  bootstrap:
+    image:
+      name: giantswarm/cluster-api-eks-bootstrap-controller
+      tag: 787579502a18152ab2c1f99b7b460e60c607b7ab
```

Comment:
I don't think we have pushed these controllers yet. We still need to do that in our fork (and make sure the watchfilter is working)